lib/python/rose/config_editor/valuewidget/array/entry.py

```diff
@@ -312,6 +317,22 @@ def populate_table(self, focus_widget=None):
                 self.set_arrow_sensitive(True, False)
         if len(self.entries) < 2:
             self.set_arrow_sensitive(False, False)
+
+        if self.has_titles:
```

Comment:
I guess you mean repeated across multiple array types rather than this and what's below? Either way, I think it is better in the *current* array implementations to have it inlined like this with the rest of the fragile code. See note at top of pull request, however, that the code in general in these should be reviewed and refactored as a followup issue.